soc/realtek/ec/Kconfig

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+if SOC_FAMILY_REALTEK_EC
+
+menuconfig REALTEK_RTS5912_IMG_HEADER
+	bool "The output binary with RTS5912 image header"
+	help
+	  the RTS5912 ROM code loads firmware image from flash to RAM by the image header setting. 
+	  Enable this to invoke the 'rts5912_imgtool' to generates the RTS5912 image header.
+
+if REALTEK_RTS5912_IMG_HEADER
+
+config REALTEK_HEADER_CHIP
+	string
+	default "RTS5912" if SOC_RTS5912	
+	
+config REALTEK_RTS5912_OUTPUT_BIN
+	bool "Build RTS5912 binary in BIN format"
+	default y
+	help
+	  Build a "raw" binary zephyr/zephyr.rts5912.bin in the build directory.
+	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
+
+config REALTEK_RTS5912_IMG_HEADER_LOAD_ADDRESS
+	string "The loading address where the RTS5912 ROM code loads firmware image from flash to RAM"
+	default "0x200003E0"
```

Comment:
string?

Reply:
Yes, it's string type parameter for image script used.

Reply:
This type should be `hex`, otherwise anyone can edit this and put in "blah blah" and it will be accepted

Reply:
Fixed